tools/stress_mcp.py

```
@@ -0,0 +1,270 @@
+
+def make_ping_frame() -> bytes:
+    return b"ping"
+
+
+def make_execute_menu_item(menu_path: str) -> bytes:
+    # Retained for manual debugging; not used in normal stress runs
+    payload = {"type": "execute_menu_item", "params": {"action": "execute", "menu_path": menu_path}}
+    return json.dumps(payload).encode("utf-8")
+
+
+async def client_loop(idx: int, host: str, port: int, stop_time: float, stats: dict):
```

Ruff (0.12.2)

Unused function argument: `idx`

(ARG001)